rules/Transform/Rector/Class_/AddAllowDynamicPropertiesAttributeRector.php

```diff
@@ -139,16 +139,18 @@ private function addAllowDynamicPropertiesAttribute(Class_ $class): Class_
 
     private function shouldSkip(Class_ $class): bool
     {
-        if ($this->transformOnNamespaces !== []) {
-            $className = (string) $this->nodeNameResolver->getName($class);
-            foreach ($this->transformOnNamespaces as $transformOnNamespace) {
-                if ($this->nodeNameResolver->isStringName($className, $transformOnNamespace)) {
-                    continue;
-                }
-
+        $className = (string) $this->nodeNameResolver->getName($class);
+        foreach ($this->transformOnNamespaces as $transformOnNamespace) {
+            if (str_contains($transformOnNamespace, '*')) {
                 if (! fnmatch($transformOnNamespace, $className, FNM_NOESCAPE)) {
                     return true;
                 }
+
+                continue;
+            }
+
+            if ($this->nodeNameResolver->isStringName($className, $transformOnNamespace)) {
+                continue;
```

Comment:
this needs to be `return true` ?

Reply:
hmm actually I am not sure. is this area covered by tests?

Reply:
I am going to add new fixture to cover this

Reply:
it's not, it need to be processed as exists